components/claim/components/email_to_ldap.jsx

```diff
@@ -58,7 +58,7 @@ export default class EmailToLDAP extends React.Component {
             return;
         }
 
-        const ldapPassword = this.ldapPassword.current.value;
+        const ldapPassword = this.ldapPasswordInput.current.value;
```

Comment:
current can be null

Reply:
@enahum Admittedly, Refs are new to me. But I don't see anywhere in our codebase where we check current for null. 
@hmhealey - Since you made the original change, can you answer this?

Reply:
`current` will only be null if the referenced component isn't being rendered, but `preSubmit` can only be called when that component is visible. Regardless, we can add a defensive check